docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<p>I think including the keywords here makes it a bit clearer, so:</p><p><code>epsilon_nondominated(results, epsilons=[0.05]*len(model.outcomes), problem=problem)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>That’s perfect!</p><p> </p><p><a href="https://peps.python.org/pep-0020/" target="_blank">Explicit is better than implicit</a> :)</p>